File: 04 Kontrollstrukturen/Pascal/Aufgabe1.pas
Program Aufgabe1;

{$codepage utf8}

Var Zahl: Integer;
    ZahlMod2: Integer;

Begin
    Write('Bitte gib eine Zahl ein: ');
    Read(Zahl);

    ZahlMod2 := Zahl Mod 2;

    If ZahlMod2 = 0 Then
    Begin
        WriteLn('Die eingegebene Zahl ist gerade.');
    End Else Begin
        WriteLn('Die eingegebene Zahl ist ungerade.');
    End;
End.

File: 04 Kontrollstrukturen/Go/Aufgabe1.go
package main

import "fmt"

var (
	zahl     int
	zahlMod2 int
)

func main() {
	fmt.Print("Bitte gib eine Zahl ein: ")
	fmt.Scan(&zahl)

	zahlMod2 = zahl % 2

	if zahlMod2 == 0 {
		fmt.Println("Die eingegebene Zahl ist gerade.")
	} else {
		fmt.Println("Die eingegebene Zahl ist ungerade.")
	}
}
